qa/tasks/ceph_test_case.py

```diff
@@ -86,6 +86,9 @@ def config_set(self, section, key, value):
        self._mon_configs_set.add((section, key))
        self.ceph_cluster.mon_manager.raw_cluster_cmd("config", "set", section, key, str(value))
 
+    def cluster_cmd(self, command: str):
+        return self.mgr_cluster.mon_manager.raw_cluster_cmd(*(command.split(" ")))
```

Comment:
This failure is related https://jenkins.ceph.com/job/ceph-pull-requests/82480/consoleFull#888207447e840cee4-f4a4-4183-81dd-42855615f2c1

Reply:
fixed